tests/dummy/app/components/exclaim-components/hbox/template.hbs

```diff
@@ -0,0 +1,3 @@
+{{#each config.children as |child|}}
+  {{yield child}}
```

Comment:
Since we know that `vbox` and `hbox` have `display: flex`, we're guaranteed that whitespace doesn't matter. For `box` (which may or may not need a better name), the whole point is that it basically follows standard browsery flow, so we have to be more careful about it not having the framework accidentally inject whitespace.